packages/matter.js/src/protocol/interaction/SubscriptionHandler.ts

```diff
@@ -855,13 +862,13 @@ export class SubscriptionHandler {
                     this.isFabricFiltered,
                 );
             }
-        } catch (e) {
-            if (StatusResponseError.is(e, StatusCode.InvalidSubscription, StatusCode.Failure)) {
+        } catch (error) {
+            if (StatusResponseError.is(error, StatusCode.InvalidSubscription, StatusCode.Failure)) {
                 logger.info(`Subscription ${this.subscriptionId} cancelled by peer.`);
                 await this.cancel(false, true);
             } else {
+                StatusResponseError.accept(error);
                 await this.cancel(false);
-                throw e;
```

Comment:
It seems like this could hide a significant amount of error information, maybe there's a more specific StatusResponseError to test for?

Reply:
I would add a logline in this case but formally (after spec) no other status error should happen then the two that are already handled 3 lines above (InvalidSubscription, Failure).